engine/verification/utils/fixtures.go

```diff
@@ -299,7 +313,7 @@ func executeCollection(
 			StartState:      startStateCommitment,
 			// TODO: include event collection hash
 			EventCollection: flow.ZeroID,
-			BlockID:         executableBlock.ID(),
+			BlockID:         blockID,
```

Comment:
Previously each chunk fixture was assigning the executable block ID. However, the executable block is an intermediary and ephemeral data object solely utilized to build the chunk. Hence, different chunks of an execution result would point to different executable blocks which would not exist once this function return. This would contradict the desired expectation that all chunks in a test fixture result should have the same block ID (as it is in the real protocol code).